moveit_ros/warehouse/warehouse/src/broadcast.cpp

```diff
@@ -91,32 +100,26 @@ int main(int argc, char** argv)
     return 2;
   }
   // Set up db
-  warehouse_ros::DatabaseConnection::Ptr conn = moveit_warehouse::loadDatabase();
+  warehouse_ros::DatabaseConnection::Ptr conn = moveit_warehouse::loadDatabase(node);
   if (vm.count("host") && vm.count("port"))
     conn->setParams(vm["host"].as<std::string>(), vm["port"].as<std::size_t>());
   if (!conn->connect())
     return 1;
 
-  ros::AsyncSpinner spinner(1);
-  spinner.start();
+  rclcpp::executors::StaticSingleThreadedExecutor executor;
+  executor.add_node(node);
```

Comment:
so - just a note - this changes the threading behavior from ROS1 - there would be 2 threads in ROS1, not sure if this leads to any blocking/deadlock at runtime.